### PR TITLE
Add icon for Viber, mamen & VLC

### PR DESCRIPTION
**Pull request by xSil3nt**
1. Viber - com.viber.voip
2. mamen - mv.com.dhiraagu.youth
3. VLC - org.videolan.vlc

### CODE REVIEW

File: app/src/main/res/drawable/mamen.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+      android:strokeWidth=".1"
+      android:strokeColor="#000"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/mamen.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M8.5049,2C6.9101,2 5.5922,3.3178 5.5922,4.9126c0,1.5948 1.3178,2.9126 2.9126,2.9126 1.5948,0 2.9126,-1.3178 2.9126,-2.9126C11.4175,3.3178 10.0997,2 8.5049,2ZM15.4951,2c-1.5948,0 -2.9126,1.3178 -2.9126,2.9126 0,1.5948 1.3178,2.9126 2.9126,2.9126 1.5948,0 2.9126,-1.3178 2.9126,-2.9126C18.4078,3.3178 17.0899,2 15.4951,2ZM8.5049,10.7379c-2.56,0 -4.6602,2.1002 -4.6602,4.6602L3.8447,21.6117A0.3883,0.3883 0,0 0,4.233 22h1.5534a0.3883,0.3883 0,0 0,0.3883 -0.3884v-6.2136c0,-1.3007 1.0294,-2.3301 2.3301,-2.3301 1.3007,0 2.3301,1.0294 2.3301,2.3301L10.835,21.6117A0.3883,0.3883 0,0 0,11.2233 22h1.5534a0.3883,0.3883 0,0 0,0.3884 -0.3884v-6.2136c0,-1.3007 1.0294,-2.3301 2.3301,-2.3301 1.3007,0 2.3301,1.0294 2.3301,2.3301L17.8252,21.6117A0.3883,0.3883 0,0 0,18.2136 22h1.5534a0.3883,0.3883 0,0 0,0.3884 -0.3884v-6.2136c0,-2.56 -2.1002,-4.6602 -4.6602,-4.6602 -1.3875,0 -2.6389,0.6178 -3.4951,1.5906 -0.8562,-0.9728 -2.1076,-1.5906 -3.4951,-1.5906z"
      android:strokeWidth="0.194175"
      android:fillColor="#000000"/>
</vector>
```

File: svgs/mamen.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
Comment (patrykmichalik) [inline svgs/mamen.svg:4]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   viewBox="0 0 24 24"
   version="1.1"
   id="svg4"
   sodipodi:docname="mamen.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="16.1875"
     inkscape:cx="0.24710425"
     inkscape:cy="13.683398"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4" />
  <path
     id="path1406"
     style="color:#000000;fill:#000000;stroke-width:0.194175;-inkscape-stroke:none"
     d="M 8.504854,2 C 6.910059,2 5.59223,3.317826 5.59223,4.912621 c 0,1.594796 1.317829,2.912622 2.912624,2.912622 1.594796,0 2.912622,-1.317826 2.912622,-2.912622 C 11.417476,3.317826 10.09965,2 8.504854,2 Z m 6.990292,0 c -1.594796,0 -2.912622,1.317826 -2.912622,2.912621 0,1.594796 1.317826,2.912622 2.912622,2.912622 1.594795,0 2.912621,-1.317826 2.912621,-2.912622 C 18.407767,3.317826 17.089941,2 15.495146,2 Z m -6.990292,8.737864 c -2.559952,0 -4.660194,2.100243 -4.660194,4.660194 V 21.61165 A 0.38834951,0.38834951 0 0 0 4.23301,22 h 1.553398 a 0.38834951,0.38834951 0 0 0 0.388349,-0.38835 v -6.213592 c 0,-1.300676 1.029422,-2.330097 2.330097,-2.330097 1.300676,0 2.330097,1.029421 2.330097,2.330097 V 21.61165 A 0.38834951,0.38834951 0 0 0 11.223301,22 h 1.553398 a 0.38834951,0.38834951 0 0 0 0.38835,-0.38835 v -6.213592 c 0,-1.300676 1.029421,-2.330097 2.330097,-2.330097 1.300675,0 2.330097,1.029421 2.330097,2.330097 V 21.61165 A 0.38834951,0.38834951 0 0 0 18.213592,22 h 1.553398 a 0.38834951,0.38834951 0 0 0 0.38835,-0.38835 v -6.213592 c 0,-2.559951 -2.100242,-4.660194 -4.660194,-4.660194 -1.387507,0 -2.638949,0.617804 -3.495146,1.590565 -0.856197,-0.972761 -2.107639,-1.590565 -3.495146,-1.590565 z" />
</svg>
```

File: svgs/viber.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
Comment (patrykmichalik) [inline svgs/viber.svg:4]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   fill="none"
   version="1.1"
   id="svg4"
   sodipodi:docname="viber.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="true"
     inkscape:zoom="22.892582"
     inkscape:cx="-1.7036086"
     inkscape:cy="13.825439"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4">
    <inkscape:grid
       type="xygrid"
       id="grid901"
       empspacing="1" />
  </sodipodi:namedview>
  <g
     id="g835">
    <path
       id="path1233"
       class="cls-1"
       d="m 11.499404,2.0015006 c -1.6043102,0.0211 -5.0540602,0.28445 -6.9828902,2.05369 -1.43159,1.43189 -1.9352,3.52708 -1.98833,6.1246604 -0.0531,2.59758 -0.12182,7.46563 4.57064,8.78558 h 0.004 l -0.003,2.01412 c 0,0 -0.03,0.81546 0.50693,0.98173 0.64948,0.20169 1.03063,-0.41811 1.65069,-1.08619 0.34034,-0.36675 0.8103602,-0.90544 1.1645102,-1.31721 3.20924,0.27011 5.67746,-0.34729 5.95778,-0.43852 0.64797,-0.21009 4.31456,-0.68008 4.91091,-5.54751 0.61556,-5.0171904 -0.29773,-8.1906704 -1.94811,-9.6210604 -0.49701,-0.45859 -2.50542,-1.91665 -6.97909,-1.93645 0,0 -0.32972,-0.0199 -0.8645,-0.0128 z m 0.0478,1.40971 c 0.4541,-0.003 0.73419,0.0152 0.73419,0.0152 3.78428,0.0165 5.59762,1.15609 6.0181,1.53785 1.39618,1.19539 2.10749,4.05587 1.58738,8.2465204 v 10e-4 c -0.50331,4.0643 -3.47728,4.31973 -4.02561,4.49561 -0.2332,0.075 -2.401,0.61432 -5.12704,0.43635 0,0 -2.0312502,2.45055 -2.6657202,3.08771 -0.099,0.0996 -0.21547,0.13986 -0.29321,0.12005 -0.10924,-0.0267 -0.13924,-0.15606 -0.13804,-0.34514 0.002,-0.27011 0.0174,-3.34698 0.0174,-3.34698 -3.96946,-1.10206 -3.73808,-5.2453 -3.69307,-7.41491 0.045,-2.1696004 0.45259,-3.9469404 1.6633,-5.1423404 1.63148,-1.47774 4.5600102,-1.68158 5.9223102,-1.69113 z m 0.31818,2.16864 a 0.25300547,0.25300547 0 0 0 -0.25476,0.25123 0.25300547,0.25300547 0 0 0 0.25124,0.25476 c 1.35271,0.009 2.45431,0.44763 3.35613,1.32676 0.8943,0.8718 1.34963,2.05602 1.36146,3.6107104 a 0.25300547,0.25300547 0 0 0 0.25493,0.25112 0.25300547,0.25300547 0 0 0 0.25112,-0.25493 c -0.0126,-1.6534604 -0.51585,-2.9958504 -1.51434,-3.9692204 -0.99097,-0.96603 -2.24332,-1.46025 -3.70578,-1.47043 z m -3.4211402,0.56959 c -0.12504,0.002 -0.25745,0.0368 -0.3925,0.10715 -0.002,0 -0.006,0.002 -0.008,0.002 a 3.8875205,3.8875205 0 0 0 -0.95682,0.77675 c -0.002,0.004 -0.004,0.004 -0.005,0.007 -0.2225,0.26851 -0.34986,0.53168 -0.38208,0.78958 a 0.50631106,0.50631106 0 0 0 -0.007,0.11525 1.0504379,1.0504379 0 0 0 0.0513,0.33553 l 0.0123,0.008 c 0.11224,0.39856 0.39314,1.06242 1.0036,2.1692804 a 12.93029,12.93029 0 0 0 1.20378,1.82209 9.5532823,9.5532823 0 0 0 0.72692,0.8196 c 0.009,0.009 0.0183,0.0186 0.0276,0.0276 l 0.0276,0.0277 0.0276,0.0276 0.0276,0.0276 a 9.5532823,9.5532823 0 0 0 0.8196602,0.72687 12.917084,12.917084 0 0 0 1.82238,1.20384 c 1.10656,0.61046 1.77102,0.89136 2.16899,1.0036 l 0.008,0.0123 a 1.0504379,1.0504379 0 0 0 0.33553,0.0516 0.52191756,0.52191756 0 0 0 0.11525,-0.007 c 0.2583,-0.0306 0.52134,-0.1578 0.78906,-0.38149 0.003,-0.002 0.003,-0.004 0.007,-0.005 a 3.895924,3.895924 0 0 0 0.77699,-0.95683 c 0,-0.002 0.002,-0.005 0.002,-0.007 0.18758,-0.36015 0.12393,-0.70166 -0.15159,-0.93426 -0.002,0 -0.58014,-0.4826 -0.86256,-0.69029 a 11.004687,11.004687 0 0 0 -0.92764,-0.59245 c -0.42557,-0.2371 -0.86016,-0.0885 -1.04023,0.14438 l -0.3734,0.46964 c -0.19148,0.2359 -0.54709,0.20593 -0.54709,0.20593 -2.59938,-0.66358 -3.2951002,-3.29657 -3.2951002,-3.29657 0,0 -0.0312,-0.3556604 0.20587,-0.5471404 l 0.4697102,-0.37334 c 0.23289,-0.17948 0.38147,-0.61496 0.14437,-1.04024 a 11.004687,11.004687 0 0 0 -0.5924502,-0.92863 c -0.20679,-0.28242 -0.68849,-0.86076 -0.69029,-0.86256 -0.14519,-0.17126 -0.33305,-0.26029 -0.54146,-0.25722 z m 3.8838102,0.74351 a 0.25300547,0.25300547 0 0 0 -0.27082,0.23389 0.25300547,0.25300547 0 0 0 0.23389,0.27076 c 0.96585,0.0707 1.68147,0.38711 2.2044,0.95793 0.52431,0.57297 0.77392,1.2692 0.75506,2.1405604 a 0.25300547,0.25300547 0 0 0 0.24748,0.25845 0.25300547,0.25300547 0 0 0 0.25839,-0.24748 c 0.0212,-0.9773604 -0.27682,-1.8257404 -0.88771,-2.4932704 a 0.25303078,0.25303078 0 0 0 -1.2e-4,-6e-5 c -0.61754,-0.6741 -1.48057,-1.0432 -2.54057,-1.12078 z m 0.37856,1.35742 a 0.25300547,0.25300547 0 0 0 -0.26584,0.23952 0.25300547,0.25300547 0 0 0 0.23952,0.26583 c 0.43605,0.0228 0.72931,0.14575 0.92839,0.35159 0.19908,0.20584 0.32149,0.51604 0.34409,0.9716604 a 0.25300547,0.25300547 0 0 0 0.26525,0.24016 0.25300547,0.25300547 0 0 0 0.24016,-0.26525 c -0.0266,-0.5366004 -0.18084,-0.9830604 -0.48577,-1.2983404 -0.30493,-0.31527 -0.74306,-0.4779 -1.2658,-0.50517 z"
       style="fill:#000000;stroke-width:0.0300125" />
  </g>
</svg>
```

File: app/src/main/res/drawable/viber.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/viber.xml:9]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="m11.4994,2.0015c-1.6043,0.0211 -5.0541,0.2844 -6.9829,2.0537 -1.4316,1.4319 -1.9352,3.5271 -1.9883,6.1247 -0.0531,2.5976 -0.1218,7.4656 4.5706,8.7856h0.004l-0.003,2.0141c0,0 -0.03,0.8155 0.5069,0.9817 0.6495,0.2017 1.0306,-0.4181 1.6507,-1.0862 0.3403,-0.3668 0.8104,-0.9054 1.1645,-1.3172 3.2092,0.2701 5.6775,-0.3473 5.9578,-0.4385 0.648,-0.2101 4.3146,-0.6801 4.9109,-5.5475 0.6156,-5.0172 -0.2977,-8.1907 -1.9481,-9.6211 -0.497,-0.4586 -2.5054,-1.9167 -6.9791,-1.9365 0,0 -0.3297,-0.0199 -0.8645,-0.0128zM11.5472,3.4112c0.4541,-0.003 0.7342,0.0152 0.7342,0.0152 3.7843,0.0165 5.5976,1.1561 6.0181,1.5379 1.3962,1.1954 2.1075,4.0559 1.5874,8.2465v0.001c-0.5033,4.0643 -3.4773,4.3197 -4.0256,4.4956 -0.2332,0.075 -2.401,0.6143 -5.127,0.4363 0,0 -2.0313,2.4506 -2.6657,3.0877 -0.099,0.0996 -0.2155,0.1399 -0.2932,0.12 -0.1092,-0.0267 -0.1392,-0.1561 -0.138,-0.3451 0.002,-0.2701 0.0174,-3.347 0.0174,-3.347 -3.9695,-1.1021 -3.7381,-5.2453 -3.6931,-7.4149 0.045,-2.1696 0.4526,-3.9469 1.6633,-5.1423 1.6315,-1.4777 4.56,-1.6816 5.9223,-1.6911zM11.8654,5.5799a0.253,0.253 0,0 0,-0.2548 0.2512,0.253 0.253,0 0,0 0.2512,0.2548c1.3527,0.009 2.4543,0.4476 3.3561,1.3268 0.8943,0.8718 1.3496,2.056 1.3615,3.6107a0.253,0.253 0,0 0,0.2549 0.2511,0.253 0.253,0 0,0 0.2511,-0.2549c-0.0126,-1.6535 -0.5159,-2.9959 -1.5143,-3.9692 -0.991,-0.966 -2.2433,-1.4603 -3.7058,-1.4704zM8.4442,6.1494c-0.125,0.002 -0.2575,0.0368 -0.3925,0.1072 -0.002,0 -0.006,0.002 -0.008,0.002a3.8875,3.8875 0,0 0,-0.9568 0.7768c-0.002,0.004 -0.004,0.004 -0.005,0.007 -0.2225,0.2685 -0.3499,0.5317 -0.3821,0.7896a0.5063,0.5063 0,0 0,-0.007 0.1152,1.0504 1.0504,0 0,0 0.0513,0.3355l0.0123,0.008c0.1122,0.3986 0.3931,1.0624 1.0036,2.1693a12.9303,12.9303 0,0 0,1.2038 1.8221,9.5533 9.5533,0 0,0 0.7269,0.8196c0.009,0.009 0.0183,0.0186 0.0276,0.0276l0.0276,0.0277 0.0276,0.0276 0.0276,0.0276a9.5533,9.5533 0,0 0,0.8197 0.7269,12.9171 12.9171,0 0,0 1.8224,1.2038c1.1066,0.6105 1.771,0.8914 2.169,1.0036l0.008,0.0123a1.0504,1.0504 0,0 0,0.3355 0.0516,0.5219 0.5219,0 0,0 0.1152,-0.007c0.2583,-0.0306 0.5213,-0.1578 0.7891,-0.3815 0.003,-0.002 0.003,-0.004 0.007,-0.005a3.8959,3.8959 0,0 0,0.777 -0.9568c0,-0.002 0.002,-0.005 0.002,-0.007 0.1876,-0.3602 0.1239,-0.7017 -0.1516,-0.9343 -0.002,0 -0.5801,-0.4826 -0.8626,-0.6903a11.0047,11.0047 0,0 0,-0.9276 -0.5925c-0.4256,-0.2371 -0.8602,-0.0885 -1.0402,0.1444l-0.3734,0.4696c-0.1915,0.2359 -0.5471,0.2059 -0.5471,0.2059 -2.5994,-0.6636 -3.2951,-3.2966 -3.2951,-3.2966 0,0 -0.0312,-0.3557 0.2059,-0.5471l0.4697,-0.3733c0.2329,-0.1795 0.3815,-0.615 0.1444,-1.0402a11.0047,11.0047 0,0 0,-0.5925 -0.9286c-0.2068,-0.2824 -0.6885,-0.8608 -0.6903,-0.8626 -0.1452,-0.1713 -0.3331,-0.2603 -0.5415,-0.2572zM12.3281,6.893a0.253,0.253 0,0 0,-0.2708 0.2339,0.253 0.253,0 0,0 0.2339,0.2708c0.9658,0.0707 1.6815,0.3871 2.2044,0.9579 0.5243,0.573 0.7739,1.2692 0.7551,2.1406a0.253,0.253 0,0 0,0.2475 0.2585,0.253 0.253,0 0,0 0.2584,-0.2475c0.0212,-0.9774 -0.2768,-1.8257 -0.8877,-2.4933a0.253,0.253 0,0 0,-0.0001 -0.0001c-0.6175,-0.6741 -1.4806,-1.0432 -2.5406,-1.1208zM12.7066,8.2504a0.253,0.253 0,0 0,-0.2658 0.2395,0.253 0.253,0 0,0 0.2395,0.2658c0.436,0.0228 0.7293,0.1458 0.9284,0.3516 0.1991,0.2058 0.3215,0.516 0.3441,0.9717a0.253,0.253 0,0 0,0.2652 0.2402,0.253 0.253,0 0,0 0.2402,-0.2652c-0.0266,-0.5366 -0.1808,-0.9831 -0.4858,-1.2983 -0.3049,-0.3153 -0.7431,-0.4779 -1.2658,-0.5052z"
      android:strokeWidth="0.0300125"
      android:fillColor="#000000"/>
</vector>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Sorry for keeping you waiting! Here are some tweaks I’d suggest.

**Comment by Chefski** [comment]
Outdated icon style. Can be closed @Goooler

**Pull request closed without merging**